llvm/include/llvm/ADT/PagedVector.h

```diff
@@ -103,6 +103,14 @@ template <typename T, size_t PageSize = 1024 / sizeof(T)> class PagedVector {
   /// Return the size of the vector.
   [[nodiscard]] size_t size() const { return Size; }
 
+  /// @return true in case the element at index @a Index belongs to a page which
+  /// was already materialised.
```

Comment:
nit: This uses a different style than all the other function documentation in this file. For consistency, shouldn't this be something like:
```suggestion
  /// Return true if the element at `Index` belongs to a page which was already
  /// materialized, i.e., had at least one element accessed.
```